### PR TITLE
Users table endpoints

### PR DESCRIPTION
**Pull request by vm2052**
Added a script to create a Users table (+ a script for privileges just in case)
Added Post, Put, Get, Delete methods for /users route according to the task description

### CODE REVIEW

File: scripts/script_user_1.sql
Comment (yellalena) [inline scripts/script_user_1.sql:1]: so this script is just supposed to be copied and pasted into psql, or is there any fancy way to run it right from the file?

**Review by yellalena — COMMENTED** [review]
Great job!!!

File: scripts/script_privileges.sql
Comment (yellalena) [inline scripts/script_privileges.sql:1]: also same note as to routes' files. we can drop the name prefix here :)

File: routes/users_routes.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    }
+});
+
+// Create a new user
+router.post("/users", async (req, res) => {
+    const { name, email, about, languages} = req.body;
+    console.log(req.body); // Log the entire request body
+    try {
+        const existingUser = await pool.query(
+            "SELECT * FROM users WHERE name = $1 OR email = $2",
```
[the remark this answers is not in the document]
Comment (vm2052) [reply routes/users_routes.js:55]: yeah I was kinda confused whether to take name as a username or we're gonna have two separate fields for that. i'll leave just name for now

**Review by yellalena — APPROVED** [review]

File: routes/applications.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    } catch (err) {
+        console.error(err);
+        res.status(500).send("Server Error");
+    }
+});
+
+
+
+router.post("/applications", async (req, res) => {
+    const { userId, text } = req.body;
```
Comment (marinacatbytes) [inline routes/applications.js:20]: Now I think we need a separate endpoint for creating application to become "member" and creating application to become "mentor". If this is for "member" then I think we won't have the userId, but we'll have other fields completed by user, i.e. firstName, lastName, email etc.

**Review by marinacatbytes — APPROVED** [review]

**Pull request merged**